src/ssh_audit/ssh_audit.py

```diff
@@ -836,82 +839,129 @@ def process_commandline(out: OutputBuffer, args: List[str], usage_cb: Callable[.
+
+        aconf.json = namespace.json
+        if namespace.json_indent is True:
+            setattr(namespace, 'json', True)
+            aconf.json = namespace.json
+            aconf.json_print_indent = namespace.json_indent
+
+        if namespace.batch is True:
             aconf.batch = True
             aconf.verbose = True
-        elif o in ('-c', '--client-audit'):
-            aconf.client_audit = True
-        elif o in ('-j', '--json'):
-            if aconf.json:  # If specified twice, enable indent printing.
-                aconf.json_print_indent = True
-            else:
-                aconf.json = True
-        elif o in ('-v', '--verbose'):
+
+        aconf.client_audit = namespace.client_audit
+
+        ttime = namespace.timeout
+        if ttime != 5:
+            aconf.timeout = float(namespace.timeout)
+            aconf.timeout_set = True
+
+        if namespace.verbose is True:
             aconf.verbose = True
             out.verbose = True
-        elif o in ('-l', '--level'):
-            if a not in ('info', 'warn', 'fail'):
-                usage_cb(out, 'level {} is not valid'.format(a))
-            aconf.level = a
-        elif o in ('-t', '--timeout'):
-            aconf.timeout = float(a)
-            aconf.timeout_set = True
-        elif o in ('-M', '--make-policy'):
+
+        # Get error level regex
+        err_level = (namespace.level)
+        pattern = re.compile(r'info|warn|fail')
+        if pattern.match(err_level):
+            aconf.level = str(namespace.level)
+        else:
+            usage_cb(out, 'Error level : {} is not valid'.format(err_level))
+
+        if getattr(namespace, 'make_policy') is True:   
             aconf.make_policy = True
-            aconf.policy_file = a
-        elif o in ('-P', '--policy'):
-            aconf.policy_file = a
-        elif o in ('-T', '--targets'):
-            aconf.target_file = a
-
-            # If we're on Windows, and we can't use the idna workaround, force only one thread to be used (otherwise a crash would occur).
-            # if no_idna_workaround:
-            #    print("\nWARNING: the idna module was not found on this system, thus only single-threaded scanning will be done (this is a workaround for this Windows-specific crash: https://github.com/python/cpython/issues/73474).  Multi-threaded scanning can be enabled by installing the idna module (pip install idna).\n")
-            #    aconf.threads = 1
-        elif o == '--threads':
-            aconf.threads = int(a)
-            # if no_idna_workaround:
-            #    aconf.threads = 1
-        elif o in ('-L', '--list-policies'):
+            aconf.policy_file = namespace.make_policy
+
+        if getattr(namespace, 'policy') is True:
+            aconf.policy_file = namespace.policy
+
+        if getattr(namespace, 'targets') is True:
+            aconf.target_file = namespace.targets
+
+        if os.name == 'nt':
+            aconf.threads = '1'
```

Comment:
I don't think restricting the number of threads is necessary for Windows platforms.